i18n/en.json

```diff
@@ -6548,6 +6548,38 @@
     "translation": "Unable to load mattermost configuration file:  Adding DefaultClientLocale to AvailableLocales."
   },
   {
+    "id": "utils.config.bucket_empty.app_error",
+    "translation": "Failed: S3 bucket empty."
+  },
+  {
+    "id": "utils.config.error_checking_bucket_exist.app_error",
+    "translation": "Error checking if bucket exist."
+  },
+  {
+    "id": "utils.config.error_creating_bucket.app_error",
+    "translation": "Error creating S3 bucket."
+  },
+  {
+    "id": "utils.config.bad_connection_to_s3_or_minio.app_error",
+    "translation": "Bad connection to S3 or minio storage."
+  },
+  {
+    "id": "utils.config.set_amazon_bucket",
+    "translation": "Set Amazon S3 Bucket to '%v'."
```

Comment:
What are the values for `%v`? (same for below strings)

Reply:
Just echoing the bucket input from the user. Regarding endpoint/region, echoing the the value based from the changes due to validation.

Reply:
Are these messages included in the logs?